packages/backend/src/models/SavedChartModel.ts

```diff
@@ -849,16 +894,31 @@ export class SavedChartModel {
                         }),
                     ),
                     additionalMetrics,
-                    customDimensions: customDimensionsRows?.map((cd) => ({
-                        id: cd.id,
-                        name: cd.name,
-                        dimensionId: cd.dimension_id,
-                        table: cd.table,
-                        binType: cd.bin_type,
-                        binNumber: cd.bin_number,
-                        binWidth: cd.bin_width,
-                        customRange: cd.custom_range,
-                    })),
+                    customDimensions: [
+                        ...(
+                            customBinDimensionsRows || []
+                        ).map<CustomBinDimension>((cd) => ({
```

Comment:
nop, thats in saved under `columnOrder` line 851

Reply:
What is the `order` field we have on the underlying table? Is order stored there or on another order table?

Reply:
tech debt shenanigans. Instead of saving the column's order in the `saved query version` table we save the order index in the fields/custom dimension/table calculation tables.